docs/azure_arc_jumpstart/azure_arc_k8s/day2/cluster_api/cluster_api_keyvault_extension/_index.md

```diff
@@ -0,0 +1,148 @@
+---
+type: docs
+title: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+linkTitle: "Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions"
+weight: 3
+description: >
+---
+
+## Integrate Azure Key Vault with Cluster API as an Azure Arc Connected Cluster using Kubernetes extensions
+
+The following README will guide you on how to enable [Azure Key Vault](https://docs.microsoft.com/en-us/azure/key-vault/general/overview) for a Cluster API that is projected as an Azure Arc connected cluster.
+
+In this guide, you will hook the Cluster API to Azure Key Vault by deploying the [Azure Key Vault extension](https://docs.microsoft.com/en-us/azure/aks/csi-secrets-store-driver) and a sample app on your Kubernetes cluster in order to integrate Azure Key Vault as a secrets store with a Kubernetes cluster via a CSI volume. This will also include deploying a Key Vault and sample secret on your Azure subscription.
+
+> **Note: This guide assumes you already deployed a Cluster API and connected it to Azure Arc. If you haven't, this repository offers you a way to do so in an automated fashion using a [Shell script](https://azurearcjumpstart.io/azure_arc_jumpstart/azure_arc_k8s/cluster_api/capi_azure/).**
```

Comment:
Should we use Kubernetes cluster deployed using Cluster API in the doc references here instead of `deployed a Cluster API`? Cluster API is a project for deploying Kubernetes cluster using declarative APIs.

Same comment for other references in the doc

Reply:
@aramase yes. This is a cross-reference to the Jumpstart scenario we support and align with the upstream team